src/Components/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import logo from "../assets/budget-it.png";
+
+const Header = () => {
+  return (
+    <header>
+      <div className="left">
+        <NavLink to="/">
+          {" "}
```

Comment:
This guy is still here

Reply:
Trying to figure out where it comes from so I can disable it.